packages/commons-test/src/setupTestContainersVitest.ts

```diff
@@ -63,18 +66,33 @@ export function setupTestContainersVitest(
 export function setupTestContainersVitest(
   readModelDbConfig?: ReadModelDbConfig,
   eventStoreConfig?: EventStoreConfig,
-  fileManagerConfig?: FileManagerConfig & S3Config & LoggerConfig
+  fileManagerConfig?: FileManagerConfig & S3Config & LoggerConfig,
+  emailManagerConfig?: EmailManagerConfig
+): {
+  readModelRepository: ReadModelRepository;
+  postgresDB: DB;
+  fileManager: FileManager;
+  emailManager: EmailManager;
+  cleanup: () => Promise<void>;
```

Comment:
I would create a dedicated override for this specific usage, that accepts only the readModelConfing and the emailManagerConfig, so you don't have to call it with two `undefined` in between the two

Reply:
Settled offline, overloads conflict, let's keep it as it is